protocols/gossipsub/src/behaviour.rs

```diff
@@ -3265,7 +3265,6 @@ where
     type ConnectionHandler = Handler;
     type ToSwarm = Event;
 
-    #[allow(deprecated)]
```

Comment:
Is this change related to this pull request?

Reply:
Yes because we use `Until` here before. I think it is a left-over.